<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1495</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Avocats</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1441</width>
      <height>651</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter </string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Ajout </string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Identifiant:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nom : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prenom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre de procès gagnés : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Numero de téléphone : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date de Naissance : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_proce">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="la_date">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>310</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ajouter">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>350</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="table_avocat_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="table_avocat">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>761</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tri">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>430</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioB_croiss">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Croissant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBdecroiss">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Décroissant</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nombre de Procèes gagnés</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Identifiant </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimer">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>561</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Suppression </string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Saisir l'identifiant de l'Avocat à supprimer : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_supp">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_supprimer">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer </string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Mettre à jour</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>671</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Mise à jour des cordonnés </string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Veuillez Entrer le nouveau Numéro de Téléphone :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nv_num">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Veuillez entrer le nouveau nombre de procès gagnés :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nv_nbr">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Veuillez Entrer l'identifiant de l'Avocat en question : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_modif">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_modifier">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Veuillez Entrer Le Nom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Veuillez Entrer Le Prenom : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Veuillez Entrer La Date de Naissance :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QTableView" name="table_rechercher">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>100</y>
        <width>1081</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Achercher">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_trouver">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>evaluation</string>
     </attribute>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Evaluation des Avocats actifs dans le tribunal comme : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="evaluation">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Débutant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moyen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Expert</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tab_evaluer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>811</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_evaluer">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Evaluer</string>
      </property>
     </widget>
     <widget class="QLabel" name="avocat_stat">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>70</y>
        <width>461</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_executer">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>520</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Executer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>your mail password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>150</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>490</y>
        <width>651</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Congés</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>401</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Donner L'identifiant de l'avocat : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date du Début du congé : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date de fin du congé :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Motif du congé : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_conge">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_debut">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_fin">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="motif">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_conge">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_avocat_conge">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>791</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1495</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_Des_Avocats">
    <property name="title">
     <string>Gestion Des Avocats</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuGestion_Des_Avocats"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
